### ISSUE TITLE
Last stop option for platforms.

### ISSUE THREAD

**Issue by SamDev-7** [jonafanho/Minecraft-Transit-Railway#170]
**Is your feature request related to a problem? Please describe.**
When a train arrives at the last stop, it does not force you to get off. The train will go to the depot and drop you off, which isn't the best solution. Ending up where you're not supposed to be in, causes confusion.

**Describe the solution you'd like**
An checkbox option in the platform menu to indicate that it is a last stop. Stations that are marked as last stop will "eject" the player onto the platform if they did not exit. You will also be unable to enter the train through the last stop. 

**Additional Information**
![image](https://user-images.githubusercontent.com/66446458/135717063-c0fc1841-30b4-4326-86f7-51651f11a92a.png)
Could be a checkbox under this menu

**Comment by BadHunyun** [comment]
Oh,the same to me.

**Comment by kettle-7** [comment]
Yes, making it an option like you sugest is  a good idea, otherwise it'd break thorough trains.

**Comment by kettle-7** [comment]
Duplicate of #207

**Comment by SamDev-7** [comment]
#207 is closed.

**Comment by SamDev-7** [comment]
Oops! Seems like #207 is closed because of the duplicate.
Reopened.
